osu.Game.Rulesets.Taiko/Mods/TaikoModMirror.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Taiko.Objects;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModMirror : Mod, IApplicableToDrawableRuleset<TaikoHitObject>
+    {
+        public override string Name => "Mirror";
+        public override string Acronym => "MR";
```

Comment:
I'm not sure I would want to name this "mirror" though. In osu! and mania the mirror mod has a strong connotation with changing your inputs. This does nothing of that sort, it only affects how you read.

To illustrate why this is bad: we have behaviours like changing the ruleset preserves mods by acronym, which seems not helpful or intentional in this instance.

I'm drawing a blank on better names at this point though.